backend/app/services/im/providers/telegram/provider.py

```
@@ -0,0 +1,346 @@
+            # Split into chunks if necessary
+            chunks = self._formatter.split_message(formatted)
+
+            for chunk in chunks:
+                try:
+                    # Try MarkdownV2 first
+                    await bot.send_message(
+                        chat_id=int(chat_id),
+                        text=chunk,
+                        parse_mode=ParseMode.MARKDOWN_V2,
+                    )
+                except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)